tedana/workflows/tedana.py

```diff
@@ -140,7 +140,7 @@ def _get_parser():
                           type=int,
                           help=('Value passed to repr(mdp.numx_rand.seed()) '
                                 'Set to an integer value for reproducible ICA results; '
-                                'otherwise, set to -1 for varying results across calls.'),
```

Comment:
So sorry, I meant that the default should be given in addition to this note about the special meaning of -1. I think we need both pieces of information, here !

Reply:
Oh, duh, that makes sense. Let me make Travis work some more.